test/WebSites/BasicWebSite/Controllers/HomeController.cs

```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult PlainView()
+        public object PlainView()
```

Comment:
I did the functional test because it was a cheap way of testing everything worked E2E, I will add a comment referencing 4960, but I don't feel its worth the effort of creating a new functional test. Both unit tests should be enough to verify that this bug was fixed, we simply had a test gap here. I'm also going to take a look to see if there is something else missing on the tests in that area.